samples/csharp_dotnetcore/12.nlp-with-luis/README.md

```diff
@@ -16,12 +16,23 @@ git clone https://github.com/Microsoft/botbuilder-samples.git
 - Update [BotConfiguration.bot](BotConfiguration.bot) file with your AppId, SubscriptionKey, Region and Version. 
     You can find this information under "Publish" tab for your LUIS application at [LUIS portal](https://www.luis.ai).  For example, for
 	https://westus.api.cognitive.microsoft.com/luis/v2.0/apps/XXXXXXXXXXXXX?subscription-key=YYYYYYYYYYYY&verbose=true&timezoneOffset=0&q= 
```

Comment:
I would expect to see this written using URI Template notation. That is:
  /apps/{LuisAppID}?subscription-key={LuisSubscriptionKey}

I think people tend to find that easier to ready that XXXXX YYYYYY.